### DISCUSSION TITLE
What is needed to drive Panels like ed113tc1

### DISCUSSION THREAD

**Discussion by DerPicknicker** [vroland/epdiy#290] [Q&A] [answered]
Hi, 

I found this really nice project - for me it’s kinda hard to get started. The mentioned panel ed113tc1 is really cheap around 35$ or another panel like the ES133UT1 or even bigger: EC312TT1 

Does your Board have support for adapters, which connectors are your currently supporting? 

Would be great if you can help me out to get started inside the eink world.. 

NOTE: Related to discussion #294 just take a look there

**Comment by martinberlin** [comment, marked as answer]
You need to check the PCB that is open source and linked to this repository. The current one has 33,34 and 40 pin FPC. 
It’s not a good idea IMHO to get started with unsupported panels just because they are cheaper. Because you don’t have the connector, there is probably not a timing waveform so the grays won’t be perfect (not important if you use only monochrome but even like that you can have only part of the display renderered)
Basically you need two things to drive any panel:
1 The right FPC connector, pinout and voltages 
2 A timings waveform

start with an easy to drive panel.
You need to make your first steps, then learn to run, and then you get a number to participate in the Marathon.

**Comment by DerPicknicker** [comment]
Hi @martinberlin ... I can totally understand your concerns. 

But for me it's ridiculous to spend 200$ while a bit smaller panel cost way less. 

Does anybody has used this panel or the ES108FC2 from you guys?

**Comment by martinberlin** [comment]
If you don’t see it listed and you have a passing adapter please try it yourself. 
You are not getting a confirmation here if no one tried it. So you need to also take the risk to try it yourself when you want to start with a non officially supported panel.
I also want to try new ones then I find time I try to make adapters and buy some. But first you need to be familiar with the component and try the supported ones. Like I commented before is a really bad idea to start with a panel that is unknown and you don’t have either the adapter or the right timings to use it.
Related to discussion #294

**Comment by DerPicknicker** [comment]
@martinberlin .. I understand you. Did you ordered the panel? The adapter looks Cool! 

It's for the 10" and 11" model right...?


Last point why I won't spend so much money on a single panel: Use of this panel is limited - yeah it looks nice but looks it 300$ nice?

**Reply by martinberlin** [reply]
> It's for the 10" and 11" model right...?

no idea. For the 10.8 yes. Don’t get too excited since I didn’t ordered any panel yet and it’s first try. Probably will take a few weeks.
My v7 boards will be ready in a week. I would recommend you to get one and a 9.7” cheap panel to try it out before the unknown 10.8”

**Reply by DerPicknicker** [reply]
I will wait until you got the adapter. Because the 9.7 inches doesn't have such a great resolution. 

So if you got any news just let me know. Then I will buy a board from you.

**Comment by martinberlin** [comment]
Of course the 9.7” has a nice resolution.
Will sell you a board but I never said that the adapter will be for sale, is open source and still needs to be tried. Just get a board and send me an email afterwards. This discussion is now closed.
Let’s open a new one after testing the adapter

**Reply by martinberlin** [reply]
BTW if you are looking for HD panels don't buy this old ones. Just spend 200 USD in https://www.waveshare.com/w/upload/b/b4/7.8inch_e-Paper_Specification.pdf 16 bit new epapers

> - yeah it looks nice but looks it 300$ nice?

Yes of course looks 200 USD nice
